### DISCUSSION TITLE
Folded heading icon

### DISCUSSION THREAD

**Discussion by aminnairi** [nvim-neorg/neorg#1275] [Q&A]
Hi :wave: 

I would like to know if it is possible to configure the folded icon for heading.

What I am refering is the icons on the right of a heading whenever it is folded.

![image](https://github.com/nvim-neorg/neorg/assets/18418459/e1deb14f-d64b-4087-981c-ee1c0ac7de18)

I have already searched in the [documentation](https://github.com/nvim-neorg/neorg/wiki/Concealer), unfortunately I'm having a hard time finding what I want and I don't know if this is allowed in the current version of Neorg.

Thank you for your answer :pray:

**Comment by max397574** [comment]
This isn't from neorg
Looks like the default fold text to me
either configure your foldtext or the filling char there
I think this could be changed with the fillchars setting (there is a helptag)

**Comment by aminnairi** [comment]
I've managed to update the fold character using this line in Lua.

```lua
opt.fillchars:append("fold:▾,foldopen:▾,foldsep:│,foldclose:▸")
```

However, I wanted to have a single character instead of multiple, so I'll need to dig this deeper.

Thank you for your answer!
